OpenConsole.sln

```diff
@@ -1029,6 +1031,21 @@ Global
 		{CA5CAD1A-B11C-4DDB-A4FE-C3AFAE9B5506}.Release|x64.Build.0 = Release|x64
 		{CA5CAD1A-B11C-4DDB-A4FE-C3AFAE9B5506}.Release|x86.ActiveCfg = Release|Win32
 		{CA5CAD1A-B11C-4DDB-A4FE-C3AFAE9B5506}.Release|x86.Build.0 = Release|Win32
+		{58A03BB2-DF5A-4B66-91A0-7EF3BA01269A}.AuditMode|ARM64.ActiveCfg = Release|ARM64

```

Comment:
@miniksa Do we want this here?

Reply:
This is actually the magic incantation that turns _off_ audit mode (check out the right hand side names)

Reply:
As long as the .Build.0 line is missing, it is fine. So it's fine.